mne/viz/epochs.py

```diff
@@ -65,71 +67,252 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         If None, defaults to `scalings=dict(eeg=1e6, grad=1e13, mag=1e15,
         eog=1e6)`.
-    cmap : matplotlib colormap | (colormap, bool) | 'interactive'
+    cmap : None | matplotlib colormap | (colormap, bool) | 'interactive'
         Colormap. If tuple, the first value indicates the colormap to use and
         the second value is a boolean defining interactivity. In interactive
         mode the colors are adjustable by clicking and dragging the colorbar
         with left and right mouse button. Left mouse button moves the scale up
         and down and right mouse button adjusts the range. Hitting space bar
         resets the scale. Up and down arrows can be used to change the
-        colormap. If 'interactive', translates to ('RdBu_r', True). Defaults to
-        'RdBu_r'.
+        colormap. If 'interactive', translates to ('RdBu_r', True).
+        If None, "RdBu_r" is used, unless the data is all positive, in which
+        case "Reds" is used.
     fig : matplotlib figure | None
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
-    axes : list of matplotlib axes | None
+    axes : list of matplotlib axes | dict of lists of matplotlib Axes | None
         List of axes instances to draw the image, erp and colorbar to.
         Must be of length three if colorbar is True (with the last list element
         being the colorbar axes) or two if colorbar is False. If both fig and
-        axes are passed an error is raised. Defaults to None.
+        axes are passed an error is raised.
+        If `groupby` is a dict, this can also be a dict of lists of axes,
+        with the keys matching. In that case, the provided axes will be used
+        for the corresponding groups.
+        Defaults to None.
     overlay_times : array-like, shape (n_epochs,) | None
         If not None the parameter is interpreted as time instants in seconds
         and is added to the image. It is typically useful to display reaction
         times. Note that it is defined with respect to the order
         of epochs such that overlay_times[0] corresponds to epochs[0].
+    combine : None | str | callable
+        If None, return one figure per pick. If not None, aggregate over
+        channels via the indicated method. If str, must be one of "mean",
+        "std" and "gfp", in which case the average, the standard deviation or
+        the GFP over channels are calculated and plotted.
+        If callable, it must accept one positional input, the data
+        in the format (n_epochs, n_channels, n_times). It must return an
+        array (n_epochs, n_times). For example:
+
+            `combine = lambda data: np.median(data, 1)`
+
+    groupby : None | str | dict
+        If not None, combine must not be None. In this case, combining happens
```

Comment:
if `groupby` is not None `combine` could default to `'mean'` for example.
Anyway - as the first sentence I'd add a short description like: `channel/sensor groups to combine.` and only then the detailed 'decision tree'.